### PR TITLE
Drop D1 support and use swarm v6.0.0 RC

### PR DESCRIPTION
**Pull request by tiyash-basu-sociomantic**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,6 +39,10 @@ jobs:
           env: DMD=2.078.3.s* F=production
         - <<: *test-matrix
           env: DMD=2.078.3.s* F=devel
+        - <<: *test-matrix
+          env: DMD=2.085.* F=production
```
Comment (matthias-wende-sociomantic) [inline .travis.yml]: Is there a reason for using 2.085 in particular and not 2.086?

File: src/fakedmq/Storage.d [hunk 2]
```diff
@@ -115,7 +115,7 @@ public struct Storage
 
         Params:
             name = channel name
-
```
Comment (matthias-wende-sociomantic) [inline src/fakedmq/Storage.d]: Just a reminder. This seems unrelated.

File: .travis.yml (hunk 1 above)
Comment (tiyash-basu-sociomantic) [reply .travis.yml]: There wasn't a specific reason. I've changed it to 2.086.*.

**Comment by tiyash-basu-sociomantic** [comment]
Updated to use swarm `v6.0.0-rc.3`.

File: src/dmqproto/node/neo/request/Pop.d [hunk 3]
```diff
@@ -3,7 +3,7 @@
     Pop request protocol.
 
     Copyright:
-        Copyright (c) 2016-2017 dunnhumby Germany GmbH. All rights reserved.
+        Copyright (c) 2016-2017 dunnhumby Germany GmbH. All rights reservthis.ed.
```
Comment (matthias-wende-sociomantic) [inline src/dmqproto/node/neo/request/Pop.d]: reservthis.ed.

File: src/dmqproto/node/neo/request/Push.d [hunk 4]
```diff
@@ -3,7 +3,7 @@
     Push request protocol.
 
     Copyright:
-        Copyright (c) 2016-2017 dunnhumby Germany GmbH. All rights reserved.
+        Copyright (c) 2016-2017 dunnhumby Germany GmbH. All rights reservthis.ed.
```
Comment (matthias-wende-sociomantic) [inline src/dmqproto/node/neo/request/Push.d]: ditto

**Review by matthias-wende-sociomantic — APPROVED** [review]
Two tiny comments :).

File: src/dmqproto/node/neo/request/Pop.d (hunk 3 above)
Comment (tiyash-basu-sociomantic) [reply src/dmqproto/node/neo/request/Pop.d]: :man_facepalming:
